src/actions/encryptDataLocally.java

```diff
@@ -0,0 +1,8 @@
+//
+// Encrypt data locally
+//
+public static void encryptDataLocally(String userDecryptionKey, String plaintext, Policy policy, MasterKeys masterKeys) {
```

Comment:
(sorry I'm not use to Java.) If Policy and MasterKeys are types: describe these types.